web/cobrands/sass/_dropzone.scss

```diff
@@ -31,10 +35,26 @@ $dropzone-background-color--full: mix($dropzone-border-color--full, $dropzone-ba
   .dz-remove,
   .dz-cancel {
     cursor: pointer;
+    font-weight: 600;
   }
 
   .dz-message u {
-    color: $dropzone-link-color;
+    padding: 0.75em 1em;
+    text-decoration: none;
+    display: inline;
+    vertical-align: baseline;
+    margin-left: 3px;
+    font-weight: 400;
+    border-radius: 4px;
+    background: $primary;
+    color: #fff;
```

Comment:
If the background is primary, the color by default should be $primary_text which should always be an inverse that displays on $primary. I see you've used this in a couple of overrides, but I think it should be the default, then any changes made as necessary.